### PR TITLE
Cloudinary

### PR DESCRIPTION
**Pull request by sashakaverina**
Hi, guys, this push commits several edits:
 - added cloudinary to store 3 (THREE) pics when loaded with products#new 
 - updated products#show page to display attached pictures
 - updated products#index page to display the first pic of the attached
 - changed seeds to reflect the fact that product pictures are NOT a picture_url anymore but product.photos.key (attached)
 please db:drop, db:create, db:migrate and db:seed to reflect it on your localhost (Nomin, please update on heroku)
Pics are taken from unsplash random so if some of them look weird (not relevant) in the first attempt, you can do it one more time)
- edited 'you might like' section - now it's random Product instances
- fixed JS for clicking and displaying product pics

### CODE REVIEW

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -45,6 +50,6 @@ def set_product
   end
```
Comment (AidanFournier) [inline app/controllers/products_controller.rb:47]: I think product_params and self.pick_random should be put under private

File: app/models/product.rb [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
   validates :description, presence: true
   validates :price, presence: true
   validates :category, presence: true
   validates :price, numericality: { greater_than: 1 }
   validates :category, inclusion: { in: CATEGORY }
+
+
+  def self.pick_random
+    self.order(Arel.sql('RANDOM()')).first(3)
+  end
```
Comment (AidanFournier) [inline app/models/product.rb:15]: Do you have the same method (self.pick_random) in both Products controller and model?

File: app/controllers/products_controller.rb (hunk 1 above)
Comment (sashakaverina) [reply app/controllers/products_controller.rb:47]: product_params are under private in products#controller
self.pick_random is a class method in product Model

File: app/models/product.rb (hunk 2 above)
Comment (sashakaverina) [reply app/models/product.rb:15]: you're right! the one in controller is not needed, deleted it. thanks!

**Pull request merged**